docs/appendix/glossary/modular-resource.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Modular Resource
+id: modular-resource
+full_link: https://docs.viam.com/modular-resources/key-concepts/#resources
```

Comment:
You use this glossary entry most of the time to replace the link to the top level page, so that should be what we link to here
```suggestion
full_link: https://docs.viam.com/modular-resources/
```